modules/api/src/main/java/org/apache/fluo/api/config/SimpleConfiguration.java

```diff
@@ -225,6 +224,34 @@ public void save(OutputStream out) {
     }
   }
 
+  /**
+   * @since 1.2.0
+   */
+  public void addProperty(String key, Boolean value) {
+    internalConfig.addProperty(key, value);
+  }
+
+  /**
+   * @since 1.2.0
+   */
+  public void addProperty(String key, Integer value) {
+    internalConfig.addProperty(key, value);
+  }
+
+  /**
+   * @since 1.2.0
+   */
+  public void addProperty(String key, Long value) {
```

Comment:
Do we need to expose addProperty()?  If so, then it should be documented how it is different from setProperty().